source/code/providers/Container_ContainerLog.h

```diff
@@ -0,0 +1,898 @@
+/* @migen@ */
```

Comment:
I presume this was completely generated from the MOF and not otherwise modified?